src/modules/hub.bicep

```
@@ -0,0 +1,75 @@
+/**
+ * Parameters
+ */
+
+@description('Name of the hub. Used to ensure unique resource names.')
```

Aligning to the spec:
```suggestion
@description('Optional. Name of the hub. Used to ensure unique resource names. Default: "finops-hub".')
```